package.json

```diff
@@ -72,8 +73,8 @@
     "build-dev": "browserify src/bundle.js -o dist/LumaGL.js",
     "lint": "uber-standard",
     "precommit": "npm run lint -s",
-    "test": "browserify test/electron.js | testron | faucet",
-    "test-node": "babel-node --include='' test/node.js | faucet",
+    "test": "babel-node --include='' test/node.js | faucet",
```

Comment:
@mikolalysenko I can change this later, but just to share my thoughts: 
I think we may end up with an evolving menu of test approaches. Naming them all and simply have "test" reference one of them by name makes it clear and easy to switch between them:
```
   "test": "npm run test-node",
   "test-node": "babel-node --include='' test/node.js | faucet",
   "test-headless": "babel-node --include='' test/webgl.js | faucet",
   "test-electron...", 
   "test-browser...", 
   ...
```
Not sure if it still makes sense to separate test-node and test-headless anymore. The idea was test-node would only test non-webl-gl dependent code, so it would always be runnable. If headless-gl is reliably installable on both mac and linux, we might be able to drop it, if not I'd consider keeping the separation for now.